### ISSUE TITLE
Plot buy and sell events on commodities graphs

### ISSUE THREAD

**Issue by lambtho12** [beancount/fava#1298]
It would be nice to add markers on the commodities graphs to show buy and sell events of the commodity.

**Comment by yagebu** [comment]
Since the prices in the commodities chart can be just be prices, which don't correspond to any trade, I guess this would have to be a new chart.

**Comment by lambtho12** [comment]
Isn't it possible to add the buy events as an overlay of some sort?

Here is an example of what I have in mind (from [portfolio performance](https://www.portfolio-performance.info/en/), which implements this functionality).

![port-perfo](https://user-images.githubusercontent.com/26206736/131332940-34a644e9-f049-4e3d-a003-73f02997cb64.png)

**Comment by yagebu** [comment]
> Isn't it possible to add the buy events as an overlay of some sort?

Yeah, I mainly meant that it has little to do with the existing chart. Two charts could be displayed overlayed of course.
